### PR TITLE
Update dependency( dependabot )

### PR DESCRIPTION
**Pull request by kcherukupalli**
## What’s changing?

Update all dependency for the solution. 

## Why?

@nac-1146

## This PR has:

- [x ] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [x ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

File: components/src/Piipan.Components.Demo/wwwroot/components/DemoModalComponent.txt [hunk 1]
```diff
@@ -19,11 +19,11 @@
 }
 
 
-<h2 class="usa-modal__heading" id="modal-1-heading">
+<h2 class="usa-modal__heading" id="DemoModalComponent-heading">
```
Comment (bgfoshay) [inline components/src/Piipan.Components.Demo/wwwroot/components/DemoModalComponent.txt:22]: Question: Not sure this file should be included in the updates?

Comment (dborgen2) [reply components/src/Piipan.Components.Demo/wwwroot/components/DemoModalComponent.txt:22]: This looks like a missed file when I made PR https://github.com/18F/piipan/pull/3591. I added unique IDs to the modals now that we had more than one modal, and this auto-generated file must never have gotten generated. Although it has nothing to do with this PR, it should be in the codebase so we should keep it.

Comment (bgfoshay) [reply components/src/Piipan.Components.Demo/wwwroot/components/DemoModalComponent.txt:22]: 👍Makes sense. Thanks!

**Review by bgfoshay — APPROVED** [review]
LGTM

**Pull request merged**